Sources/Internal/FrameProvider.swift

```diff
@@ -152,6 +164,17 @@ final class FrameProvider {
     return CGRect(origin: CGPoint(x: x, y: y), size: daySize)
   }
 
+  func frameOfMonthFooter(_ month: Month, inMonthWithOrigin monthOrigin: CGPoint) -> CGRect {
+    let x = monthOrigin.x
+    let y = monthOrigin.y +
+      monthHeaderHeight +
+      content.monthDayInsets.top +
+      (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) +
+      ((CGFloat(numberOfRows(in: month)) * daySize.height) + (CGFloat(numberOfRows(in: month) - 1) * content.verticalDayMargin)) +
+      content.monthDayInsets.bottom
+      return CGRect(x: x, y: y, width: monthWidth, height: monthFooterHeight ?? 0)
+  }
```

Comment:
```suggestion
  func frameOfMonthFooter(_ month: Month, inMonthWithOrigin monthOrigin: CGPoint) -> CGRect {
    let numberOfWeekRows = self.numberOfWeekRows(in: month)

    let x = monthOrigin.x
    let y = monthOrigin.y +
      monthHeaderHeight +
      content.monthDayInsets.top +
      heightOfDaysOfTheWeekRowInMonth() +
      heightOfDayContent(forNumberOfWeekRows: numberOfWeekRows) +
      content.monthDayInsets.bottom
      return CGRect(x: x, y: y, width: monthWidth, height: monthFooterHeight ?? 0)
  }
```

Adopting new helpers